homeassistant/components/fritz/common.py

```diff
@@ -429,7 +429,7 @@ async def async_scan_devices(self, now: datetime | None = None) -> None:
                     self.fritz_hosts.get_mesh_topology
                 )
             ):
-                raise Exception("Mesh supported but empty topology reported")
+                raise HomeAssistantError("Mesh supported but empty topology reported")
```

Comment:
Code owner should explain what this means and what can be done about it.

Reply:
Reverted the change and added a `pylint: disable` for now.